app/decorators/opencensus_decorators.py

```diff
@@ -0,0 +1,12 @@
+from functools import wraps
+from opencensus.trace import execution_context
+
+
+def capture_trace(f):
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        tracer = execution_context.get_opencensus_tracer()
+        with tracer.span(name=f.__name__):
+            return f(*args, **kwargs)
```

Comment:
this is really petty but can we make `f` `function` or something like that? I have a real aversion to abbreviations in code